inst/extdata/emissions/GFED/GFED_ForestFire_SO2.csv

```diff
@@ -0,0 +1,250 @@
+USA,0,26010.75442,11386.69979,20996.91812,26389.56054,17288.9032,7335.300792,54311.49963,24902.46794,157251.2236,107957.9267,20555.60555,39266.98995,21489.99254,87359.65758,19188.49546,18401.47435,29992.84838,39671.41223
+BFA,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+URY,0,0.157503461,0.023802784,0.033114286,0.025710864,0.546893542,0,0,0.008515212,0,0.135691418,0.100962658,0.006767053,0.009597319,0,0,0,0,0
+UZB,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+VCT,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+VEN,0,248.7468805,159.7816369,131.2478623,112.0052552,146.819867,416.9232947,33.23311,651.025493,94.37402181,113.3000155,97.61011738,321.0797005,169.4047332,64.29614643,290.189772,11.98071553,6.70301817,124.5664599
+VGB,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+VNM,0,88.56090645,30.25440987,121.4910721,49.31439594,30.48960902,1.756751882,8.752814413,39.70328792,125.9368204,56.16213048,27.40824957,284.7455749,3.110293599,82.56061183,188.9313743,55.25405203,100.3142671,33.23137962
+VIR,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+NAM,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+WLF,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+WSM,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+SWZ,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+YEM,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+ZMB,0,3.309974935,15.88546261,21.61571227,21.00632662,15.8948977,21.40057102,0,0,0,0,62.33283831,8.309663919,17.11019483,11.54264367,20.34344719,38.00160417,0,25.53438226
+ZWE,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+IDN,0,4.810791927,823.0991475,26.68754849,76.48421741,18.29383446,52.60939325,184.1835391,24.0324067,89.02366182,54.17924421,124.2137549,6.778476735,0.871359366,133.8286006,0.876768376,25.73213898,38.79223421,4.760510118
+GLP,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+ANT,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+ARE,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+TLS,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+PCN,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+PLW,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+MHL,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+SPM,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+SHN,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+SMR,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+TCA,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+ESH,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+SRB,0,11.31898448,22.26929768,19.09049345,11.65726668,63.19668878,6.925200952,12.13260602,12.7300322,3.835857547,12.89006294,3.556395415,74.27906844,11.02049892,8.786859973,13.80372062,24.02549792,47.11094834,10.44568533
+SJM,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+MAF,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+BLM,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+GGY,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+JEY,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+SGS,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+TWN,0,31.30683823,29.99949166,26.95520487,16.71974583,11.67667501,29.38038453,14.97892603,29.41479104,26.880287,12.56514087,46.17905115,61.87773397,41.74213486,28.87182159,20.99689662,23.58092513,18.98013807,35.87890212
```

Comment:
I suspect we're going to write a script at the end of this process to read in all data system inputs and write them back out again, to ensure line endings and final newlines are all fixes.